docs/upgrade-guide/upgrade211-30.md

```diff
@@ -0,0 +1,167 @@
+To update ANGULAR version from 11.0 to 12.0 by running the below update commands
+
+```sh
+ng update @angular/core@12 @angular/cli@12 –force
+npm i
+```
+    
+Below are the some updates when you upgrade to v12.0
+
+    • ANGULAR now requires TypeScript 4.2. ng update will update you automatically.
+    
+    • IE11 support has been deprecated.
+    
+    • You can no longer use ANGULAR with Node.js version 10 or older.
+    
+    • Update zone.js to version 0.11.4. ng update will update this dependency automatically.
+    
+    • Remove 'emitDecoratorMetadata' TypeScript compiler option.
+    
+    • Remove deprecated ViewEngine-based i18n build and extract options.
+    
+    • Updates Web Worker consumer usage to use the new syntax supported directly by Webpack 5.
+    
+    • `XhrFactory` has been moved from `@angular/common/http` to `@angular/common`.
+
+
+
+**Upgrade to v13:**
+
+To update ANGULAR version from 12.0 to 13.0 by running the below update commands
+
+```sh
+ng update @angular/core@13 @angular/cli@13 –force
+npm i
+```
+    
+Below are the some updates when you upgrade to v13.0
+
+    • ANGULAR now uses TypeScript 4.4.
+    
+    • Make sure you are using Node 12.20.0 or later
+    
+    • Use [routerLink]="" in templates to [routerLink]="[]" because these links are likely intended to route to the current page with updated fragment/query params.
+    
+    • As of ANGULAR version 13, `entryComponents` are no longer necessary.
+    
+    • Remove polyfills required only for Internet Explorer which is no longer supported.
+    
+    • Remove no longer valid ANGULAR schematic options from `angular.json`.
+
+
+
+**Upgrade to v14:**
+
+To update ANGULAR version from 13.0 to 14.0 by running the below update commands
+
+```sh
+ng update @angular/core@14 @angular/cli@14 –force
+npm i
+```
+    
+Below are the some updates when you upgrade to v14.0
+
+    • ANGULAR now uses TypeScript 4.6
+    
+    • Make sure you are using Node 14.15.0 or later
+    
+    • Update ANGULAR packages 'dependencies' and 'devDependencies' version prefix to '^' instead of '~'.
+    
+    • Remove 'package.json' files from library projects secondary entrypoints.
+    
+    • Update TypeScript compilation target to 'ES2020'.
+
+
+Make sure you have install dev dependency for the below dependecies
+
+inquirer, winston
+
+```sh
+npm install --save-dev inquirer@^8.0.0
+npm install --save-dev winston@^3.8.2
+```
+
+BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
+
+This is no longer the case. Verify if you need this module and configure a polyfill for it.
+
+If you want to include a polyfill, you need to:
+
+	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
+    
+	- install 'path-browserify'
+    
+If you don't want to include a polyfill, you can use an empty module like this:
+
+	resolve.fallback: { "path": false }
+
+
+
+**Need to update:**
+
+@ngrx/router-store    			        ^14.0.0
```

Comment:
can you please format this as a table, without spacing... you can see in the PR review it is displayed odd